cypress/fixtures/commonComponents/alert.json

```diff
@@ -3,22 +3,28 @@
     
   },
   "titleOtherLanguage": {
-    "title": "mp150ú¿¡üßä"
+    "title": "",
+    "titleSpecialCharacters": "otherLanguage"
   },
   "titleSpecialCharacter": {
-    "title": "!@#$%^*()_+-=~[];:.,?{}&\"'<>"
+    "title": "",
```

Comment:
Can we not just replace `"title": "!@#$%^*()_+-=~[];:.,?{}&\"'<>"` with `"title": "specialCharacters"` rather than have an empty string and new key/value pair?

Reply:
No, because we are not able to pass special characters in url (In order to protect against XSS attacks, keys and values of args specified through the URL are limited to alphanumeric characters, spaces, underscores and dashes).
https://storybook.js.org/docs/react/writing-stories/args#setting-args-through-the-url